packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -0,0 +1,855 @@
+
+* Supports pagination (to a maximum of 200 results).
+* Requires [URL encoding](http://en.wikipedia.org/wiki/Percent-encoding). For example, `search=profile.displayName eq "Bob"` is encoded as `search=profile.displayName%20eq%20%22Bob%22`. Examples use cURL-style escaping instead of URL encoding to make them easier to read.
+
+Searches include all Device profile properties, as well as the Device `id`, `status` and `lastUpdated` properties.
+
+| Search term example                             | Description                                      |
+| :---------------------------------------------- | :----------------------------------------------- |
+| `status eq "ACTIVE"`                            | Devices that have a `status` of `ACTIVE`         |
+| `lastUpdated gt "yyyy-MM-dd'T'HH:mm:ss.SSSZ"`   | Devices last updated after a specific timestamp  |
+| `id eq "guo4a5u7JHHhjXrMK0g4"`                  | Devices with a specified `id`                    |
+| `profile.displayName eq "Bob"`                  | Devices that have a `displayName` of `Bob`       |
+| `profile.platform eq "WINDOWS"`                 | Devices that have an `platform` of `WINDOWS`     |
+| `profile.sid sw "S-1" `                         | Devices whose `sid` starts with `S-1`            |
+
+#### Permitted OAuth2 scopes 
+`okta.devices.read`
+
+#### Request path parameters
+
+None
+
+#### Request query parameters
+
+| Parameter   | Type   | Description                                                                                                               |
+| ----------- | ------ | ------------------------------------------------------------------------------------------------------------------------- |
+| `search`    | String | Searches for devices with a supported [filtering](/docs/reference/core-okta-api/#filter) expression for most properties |
+| `limit`     | Number | Specifies the number of results returned (maximum `200`)                                                                    |
+| `after`     | String | Specifies the pagination cursor for the next page of devices                                                              |
+
+* If you don't specify a value for `limit`, the maximum (200) is used as a default.
+* Treat the `after` cursor as an opaque value and obtain it through the next link relation. See [Pagination](/docs/reference/core-okta-api/#pagination).
+
+#### Request body
+
+None
+
+#### Response body
+
+Array of [Device](#device-object) objects.
+
+#### Usage example (list all Devices)
+
+The following request returns a list of all available devices, without any query parameters.
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/devices"
+```
+
+##### Response
+
+```json
+HTTP/1.1 200 OK
+Content-Type: application/json
+Link: <https://{yourOktaDomain}/api/v1/devices?limit=200>; rel="self"
```

Comment:
I don't know why `{yourOktaDomain}`is flagged as red here?

Reply:
1. This is okay to be RED as it's due to presence of tags `<>`
2. We should only add `${yourOktaDomain}` variable when it needs to be replaced by the developer when making an API call
3. Everything in the response is a placeholder for what server returns and here it represents developers okta domain.